src/components/CurriculumComponents/OakComponentsKitchen/AcceptTerms/index.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  OakCheckBox,
+  OakFieldError,
+  OakFlex,
+} from "@oaknational/oak-components";
+
+import Terms from "../Terms";
+
+import Box from "@/components/SharedComponents/Box";
+import BrushBorders from "@/components/SharedComponents/SpriteSheet/BrushSvgs/BrushBorders";
```

Comment:
I think the `TermsAgreementForm` already exists in TeacherComponents, but please ignore me if it's a different form!

Reply:
Yeah so there are multiple copies in the codebase, they just aren't very single purpose. So terms exists in both

 - `./src/components/GenericPagesComponents/NewsletterFormWrap/NewsletterFormWrap.tsx`
 - `./src/components/TeacherComponents/TermsAgreementForm/TermsAgreementForm.tsx`

But they are both not something we can use. I'm hoping we can submit PRs to use our components instead and them move this upstream into `oak-components` from out kitchen. Make sense?